crates/stdarch-verify/tests/arm.rs

```diff
@@ -394,6 +394,7 @@ fn verify_all_signatures() {
             "brk" | "__breakpoint" | "udf" | "_prefetch" => continue,
             _ => {}
         }
+        let skip_intrinsic_verify = ["vmov_n_p64", "vmovq_n_p64"];
```

Comment:
Add a small comment explaining that these are present in GCC and Clang but are missing from the official documentation.

Reply:
> Add a small comment explaining that these are present in GCC and Clang but are missing from the official documentation.

Done. Thanks.